### ISSUE TITLE
Server-sitemap is not add to main sitemap

### ISSUE THREAD

**Issue by PhilipeGatis** [iamvishnusankar/next-sitemap#361]
**Describe the bug**
When I generate the sitemap, the main/index sitemap doesn't have the dynamic server sitemap.

**To Reproduce**
Add a dynamic server sitemap and just run next-sitemap
you will see that the dynamic sitemap is not listed on the main file.

**Expected behavior**
All sitemaps references must be included on main sitemap.

**Screenshots**
the main sitemap result example
<img width="788" alt="image" src="https://user-images.githubusercontent.com/4982462/162435215-b05c1d5f-1167-4f6a-a042-32f88e39323b.png">

expected result
<img width="579" alt="image" src="https://user-images.githubusercontent.com/4982462/162435823-b148dbff-63ed-42df-8a50-440b05bcc975.png">

**Additional context**
my config from next-sitemap.js
```
module.exports = {
    siteUrl: process.env.NEXT_PUBLIC_URL,
    generateRobotsTxt: true,
    sitemapSize: 7000,
    exclude: [
        '/server-sitemap.xml',
        '/project/*',
        '/profile/*',
        '/discover',
        '/messages',
        '/notifications',
        '/onboarding',
        '/search',
        '/settings',
        '/theframe'
    ],
    robotsTxtOptions: {
        additionalSitemaps: [
            `${process.env.NEXT_PUBLIC_URL}/server-sitemap.xml`
        ],
    },
}

**Comment by lionep** [comment]
Hello, nice project here !
I have the same behavior. @PhilipeGatis did you found any workaround ?

**Comment by PhilipeGatis** [comment]
> Hello, nice project here !
> I have the same behavior. @PhilipeGatis did you found any workaround ?

No :(

**Comment by spiderhands** [comment]
@PhilipeGatis your **expected result** is not valid. I suppose you want the following result:
```
<?xml version="1.0" encoding="UTF-8"?>
<sitemapindex xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
  <sitemap>
    <loc>http://localhost:3000/sitemap-0.xml</loc>
  </sitemap>
  <sitemap>
    <loc>http://localhost:3000/server-sitemap.xml</loc>
  </sitemap>
</sitemapindex>
```

**Comment by spiderhands** [comment]
@PhilipeGatis and @lionep Here is a workaround.

```
// next-sitemap.xml

module.exports = {
  siteUrl: process.env.NEXT_PUBLIC_BASE_URL || process.env.NEXT_PUBLIC_VERCEL_URL,
  exclude: ['/server-sitemap.xml'],
}

```

```
// add-sitemap-index.js
const fs = require('fs');
const sitemapConfig = require('./next-sitemap.js');

const additionalSitemaps = [
  '/server-sitemap.xml',
].map(sitemap => sitemapConfig.siteUrl + sitemap)
  .map(sitemapUrl => `<sitemap><loc>${sitemapUrl}</loc></sitemap>`)
  .join('\n');

const sitemapFilePath = `${__dirname}/public/sitemap.xml`;
const sitemapClosingTag = '</sitemapindex>';

try {
  const sitemapContent = fs.readFileSync(sitemapFilePath, 'utf8');
  const updatedSitemapContent = sitemapContent.replace(
    sitemapClosingTag,
    `${additionalSitemaps}\n${sitemapClosingTag}`,
  );
  fs.writeFileSync(sitemapFilePath, updatedSitemapContent, 'utf8');
} catch (e) {
  console.error(e);
}
```

```
// package.json
...
"scripts": {
    "build": "next build",
    "postbuild": "npm run sitemap:build && npm run sitemap:add-index",
    "sitemap:build": "cross-env next-sitemap --config next-sitemap.js",
    "sitemap:add-index": "cross-env node add-sitemap-index.js"
}
...
```

**Comment by ahmettahasakar** [comment]
Is there going to be a fix for this? Rather than a workaround?

**Comment by BrunoElo** [comment]
Any update on this please?

**Comment by Pnlvfx** [comment]
solution?

**Comment by iamvishnusankar** [comment]
@PhilipeGatis @lionep @spiderhands @ahmettahasakar @BrunoElo @Pnlvfx 

Fixed via #400
